libbeat/common/flowhash/communityid_fips.go

```diff
@@ -0,0 +1,24 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//go:build requirefips
+
+package flowhash
+
+func hashFlow(flow Flow) string {
+	return ""
+}
```

Comment:
What do you think about moving the whole `flowhash` functionality to `nofips` and provide a noop here in a _non-breaking_ way: 
```
// CommunityID is not available in FIPS mode.
var CommunityID = NewCommunityID()

type communityIDHasher struct{}

// NewCommunityID allows to instantiate a flow hasher with custom settings.
func NewCommunityID() Hasher {
	return &communityIDHasher{}
}

// Hash returns the hash for the given flow.
func (h *communityIDHasher) Hash(flow Flow) string {
	return ""
}

```

I think this should allow to keep the changes contained in the `flowhash` package without having to touch the beats. (haven't tested it out so might be overlooking something - keep me honest).